### PR TITLE
feat: Update share on x and farcaster number of mint

### LINKED ISSUE #9727: Update share on x and farcaster number of mint
Let's add to the success modal share which # you've minted out of total supply, i.e. coinbase inspiration

image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/9705#issuecomment-1992004100_

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Feature

  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #9727

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [ ] My fix has changed UI

image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

**Comment by prury** [comment]
@yangwao can you test this one? for some reason farcaster app is kind of blocked here in Brasil, tried to access several times without success

**Review by daiagi — APPROVED** [review]

**Comment by yangwao** [comment]
> is kind of blocked here in Brasil

what error it shows you? It should be most global available social network

**Comment by yangwao** [comment]
Generally ok, it worked, yet we need to tune more actionable text and better signalling

The first thing which comes to my mind is to mention "generative art"
> I've minted a generative art piece on KodaDot #97/255 

@Jarsen136 I think can change text too, to "I've minted a generative art piece on Koda #97/255"

---
https://warpcast.com/yangwao/0xaef14f4f

image

**Comment by prury** [comment]
> > is kind of blocked here in Brasil
> 
> what error it shows you? It should be most global available social network

image

**Comment by yangwao** [comment]
@prury open from warpcast.com from desktop - https://warpcast.com/~/gift-invite?id=018e37b2-a129-0882-acd0-f6852e0dd6fc

**Comment by prury** [comment]
> @prury open from warpcast.com from desktop - https://warpcast.com/~/gift-invite?id=018e37b2-a129-0882-acd0-f6852e0dd6fc

ohhhh, thank you!! registering!

**Comment by Jarsen136** [comment]
> 
> @Jarsen136 I think can change text too, to "I've minted a generative art piece on Koda #97/255"
> 

✅ Done

**Comment by prury** [comment]
maybe add the /koda since its how things work there? 
example: 
image

how it is right now: 
image

**Comment by Jarsen136** [comment]
> maybe add the /koda since its how things work there? example: image
> 


`I've minted a generative art piece on Koda #1/1000`

Currently, the sharing text is the same across every social platform( x / telegram / farcaster). Should it be changed to a different one on Farcaster? wdyt @yangwao

File: locales/en.json [hunk 1]
```diff
@@ -1070,6 +1070,7 @@
     "boughtNft": "Check out this awesome NFT I bought on KodaDot",
     "boughtNfts": "Check out these awesome NFTs I bought on KodaDot:\n {0}",
     "nftWithArtist": "Check out this awesome NFT on {'@'}KodaDot from {'@'}{0}",
+    "dropNft": "I've minted a generative art piece on Koda {'#'}{0}{'/'}{1}",
```
Comment (prury) [inline locales/en.json:1073]:
```suggestion
    "dropNft": "I've minted a generative art piece on /Koda {'#'}{0}{'/'}{1}",
```

Comment (prury) [reply locales/en.json:1073]: i meant only this

**Comment by yangwao** [comment]
> Should it be changed to a different one on Farcaster? wdyt @yangwao

ah could be done in follow up tho, this is already great tho! TYSM!

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 11.75 USD/DOT ~ 2.553 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0xa9b33bbc106e29b029996399715e540f249a026c95fbb12136d78fde3a38849a](https://polkadot.subscan.io/extrinsic/0xa9b33bbc106e29b029996399715e540f249a026c95fbb12136d78fde3a38849a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**